website/docs/r/muting_schedule.html.markdown

```diff
@@ -0,0 +1,120 @@
+	type = "MutingSchedulesLibraryMutingSchedule"
+	content_type = "MutingSchedule"
+	monitor {
+		all = true
+	  }
+	schedule  {
+		timezone = "America/Los_Angeles"
+		start_date = "2023-08-05"
+		start_time = "00:00"
+		duration = 40
+	}
+}
+```
+
+## Example One Time Muting Schedule For Specifc Monitor/Folder ids
+
+```hcl
+resource "sumologic_muting_schedule" "muting_schedule" {
+  name = "Muting Schedule For one time"
+  description = "This is an example for one time Muting schedule for all monitor"
+	type = "MutingSchedulesLibraryMutingSchedule"
+	content_type = "MutingSchedule"
+	monitor {
+		ids = ["0000000000200B92"]
+	  }
+	schedule  {
+		timezone = "America/Los_Angeles"
+		start_date = "2023-08-05"
+		start_time = "00:00"
+		duration = 40
+	}
+}
+```
+
+## Example RRule Muting Schedule For Specifc Monitor/Folder ids
+
+```hcl
+resource "sumologic_muting_schedule" "muting_schedule" {
+  name = "Muting Schedule For one time"
+  description = "This is an example for one time Muting schedule for all monitor"
+	type = "MutingSchedulesLibraryMutingSchedule"
+	content_type = "MutingSchedule"
+	monitor {
+		all = true
+	  }
+	schedule  {
+		timezone = "America/Los_Angeles"
+		start_date = "2023-08-05"
+		start_time = "00:00"
+		duration = 40
+    rrule = "FREQ=DAILY;INTERVAL=1;BYHOUR=9,10"
+    is_form = false
+	}
+}
+```
+
+## Example RRule Muting Schedule For All monitor
+
+```hcl
+resource "sumologic_muting_schedule" "muting_schedule" {
+  name = "Muting Schedule For one time"
+  description = "This is an example for one time Muting schedule for all monitor"
+	type = "MutingSchedulesLibraryMutingSchedule"
+	content_type = "MutingSchedule"
+	monitor {
+		ids = ["0000000000200B92"]
+	  }
+	schedule  {
+		timezone = "America/Los_Angeles"
+		start_date = "2023-08-05"
+		start_time = "00:00"
+		duration = 40
+    rrule = "FREQ=DAILY;INTERVAL=1;BYHOUR=9,10"
+    is_form = false
+	}
+}
+```
+
+## Argument reference
+
+The following arguments are supported:
+
+- `type` - (Optional) The type of object model. Valid value:
+  - `MutingSchedulesLibraryMutingSchedule`
+- `name` - (Required) The name of the muting schedule. The name must be alphanumeric.
+- `description` - (Optional) The description of the muting schedule.
+- `content_type` - (Optional) The type of the content object. Valid value:
+  - `MutingSchedule`
+- `monitor` - (Required) The monitors which need to put in the muting schedule. see `monitor_scope_type`:
+- `schedule` - (Required) The schedule information. see `schedule_type`.
+
+#### schedule_type
+  - `timezone` - (Required) Time zone for the schedule per
+            [IANA Time Zone Database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List).
+  - `start_date` - (Required) Schedule start date in the format of `yyyy-mm-dd`
+  - `start_time` - (Required) Schedule start time in the format of `hh:mm`
+  - `duration` - (Required) Duration of the muting in minutes
+  - `rrule` - (Optional) RRule (Recurrence Rule)
```

Comment:
I think it would be good to provide the reference here like we provide in UI for rrule

Reply:
updated. here is the doc review result. 
![Uploading Screenshot 2023-08-15 at 6.05.43 PM.png…]()